src/main/kotlin/builder/html/HtmlBuilder.kt

```diff
@@ -0,0 +1,31 @@
+package com.htmlbuilder.builder.html
+
+import com.htmlbuilder.builder.Builder
+import com.htmlbuilder.model.dto.ContentBlock
+import com.htmlbuilder.model.dto.ContentType
+
+private const val CONTENT_MARK = "{\$content}"
+
+class HtmlBuilder : Builder {
+
+    override fun build(contentBlocks: List<ContentBlock>): String {
+        val contentToInsert =
+            contentBlocks.joinToString("\n", "\n", "\n")
+            { getOpenTag(it.type) + it.content + getCloseTag(it.type) }
+
```

Comment:
Suggestion: it's better to use the file api here. 
`Paths.of().file.readText()`

Reply:
discussed